src/pages/About/index.tsx

```diff
@@ -0,0 +1,240 @@
+import coinbaseWalletIcon from 'assets/images/coinbaseWalletIcon.svg'
+import metaMaskIcon from 'assets/images/metamask.png'
+import phantomIcon from 'assets/images/phantom.png'
+import rainbowIcon from 'assets/images/rainbow.png'
+import walletConnectIcon from 'assets/images/walletConnectIcon.svg'
+import { ButtonOutlined } from 'components/Button'
+import { Box } from 'nft/components/Box'
+import { DiscordIconMenu, GithubIconMenu, TwitterIconMenu } from 'nft/components/icons'
+import { ReactNode } from 'react'
+import { useIsDarkMode } from 'state/user/hooks'
+import styled, { useTheme } from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+
+import Card from './Card'
+import Step from './Step'
+import { SubTitle, Title } from './Title'
+
+const MOBILE_BREAKPOINT = BREAKPOINTS.md
+
+const IconRow = styled.span`
+  display: flex;
+  flex-direction: row;
+`
+
+const Icon = ({ href, children }: { href?: string; children: ReactNode }) => {
+  return (
+    <Box
+      as={href ? 'a' : 'div'}
+      href={href ?? undefined}
+      target={href ? '_blank' : undefined}
+      rel={href ? 'noopener noreferrer' : undefined}
+      display="flex"
+      flexDirection="column"
+      color="textPrimary"
+      background="none"
+      border="none"
+      justifyContent="center"
+      textAlign="center"
+      marginRight="12"
+    >
+      {children}
+    </Box>
+  )
+}
+
+const Page = styled.span<{ isDarkMode: boolean }>`
+  width: 100%;
+  align-self: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`
+
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 120px;
+  justify-content: space-between;
+  color: ${({ theme }) => theme.textPrimary};
+  font-size: 24px;
+  line-height: 36px;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}px) {
+    flex-direction: row;
+  }
+
+  & > * {
+    flex: 1;
+  }
+`
+
+const Content = styled.div`
+  max-width: 1280px;
+  pointer-events: all;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 8rem 1rem 1rem 1rem;
+  gap: 24px;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}px) {
+    padding: 8rem 5rem 5rem 5rem;
```

Comment:
pref px or em, since rem is more context-dependent and is therefore harder to reason about w/o knowledge of the parent element font size

Reply:
yea this way a copy paste, i'll change it out